hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/sigs/order/SigRequirements.java

```diff
@@ -1392,4 +1416,8 @@ private <T> SigningOrderResult<T> topicDelete(
         }
         return factory.forValidOrder(required);
     }
+
+    public SigMetadataLookup getSigMetaLookup() {
```

Comment:
add NonNull / Nullable annotation for return value

Reply:
Ooh, actually this is dead code from a previous iteration and is no longer needed. Good catch, thanks! Removed it now